styles/styles.css

```diff
@@ -0,0 +1,138 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans|Ubuntu');
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: Ubuntu, Helvetica, sans-serif;
+}
+
+a {
+  color: #0A1612;
+  text-decoration: none;
+  border-bottom: dotted;
+  border-width: thin;
+  border-color: #f7ce3e
+}
+a:hover{
+  /*color: #f7ce3e;*/
+  background-color: #f0f0f0;
+  border: none;
+}
+
+p, li {
+  font-family: 'Open Sans', Helvetica, sans-serif;
+  color: #0A1612;
+  /*text-decoration: none;*/
+}
+
+header {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding: 0 3% 1% 0;
+  background-color: #f0f0f0;
+  color: #1a2930;
+}
+
+header h1 {
+  padding-left: 6%;
+}
+
+.hello, .disclaim {
```

Comment:
It seems as if you are using the `.hello` class to style the title of your site. Might you style the `<h1>` on each page directly instead?